app/assets/javascripts/modules/nested-facets.js

```diff
@@ -65,6 +65,9 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     for (let o = 0; o < optionElements.length; o++) {
       const mainFacetValue = optionElements[o].getAttribute('data-main-facet-value')
 
+      const mainFacetLabel = optionElements[o].getAttribute('data-main-facet-label') + ' - '
```

Comment:
Nitpick: I'd suggest a name of `mainFacetLabelPrefix` instead (better implies that there is a " - " appended to the end of it)